Fabric.Terminology.Domain/DomainExtensions.String.cs

```diff
@@ -1,13 +1,15 @@
 ﻿namespace Fabric.Terminology.Domain
 {
+    using NullGuard;
+
     /// <summary>
     /// Extension methods for <see>
     ///         <cref>System.String</cref>
     ///     </see>
     /// </summary>
-    public static partial class Extensions
+    public static partial class DomainExtensions
     {
-        public static bool IsNullOrWhiteSpace(this string value)
+        public static bool IsNullOrWhiteSpace([AllowNull] this string value)
```

Comment:
`[CanBeNull]`

Reply:
Tests fail with `[CanBeNull]` but not with `[AllowNull]`.  I've added both.

Reply:
Yeah, I meant to do both. `[CanBeNull]` will clue ReSharper in that it can warn you if you forget to check the value before accessing it. Also, if you use the Implicit Nullability ReSharper extension, it will assume that any method parameters in your own code base are `[NotNull]` by default, so marking things as `[CanBeNull]` can avoid incorrect warnings in places where this method is called.